promql/promqltest/testdata/functions.test

```diff
@@ -259,6 +259,7 @@ load 5m
 	testcounter_reset_middle_mix	0+10x4 0+10x5 {{schema:0 sum:1 count:1}} {{schema:1 sum:2 count:2}}
 	http_requests_mix{job="app-server", instance="1", group="canary"}		0+80x10 {{schema:0 sum:1 count:1}}
     http_requests_histogram{job="app-server", instance="1", group="canary"}		{{schema:0 sum:1 count:2}}x10
```

Comment:
Formatting nit: Here is a (pre-existing) mix of tabs and spaces leading to bad alignment depending on tab size.